board/piksiv3/rootfs-overlay/etc/init.d/group

```diff
@@ -0,0 +1,27 @@
+root:x:0:
+daemon:x:1:
+bin:x:2:
+sys:x:3:
+adm:x:4:
+tty:x:5:
+disk:x:6:
+lp:x:7:
+mail:x:8:
+kmem:x:9:
+wheel:x:10:root
+cdrom:x:11:
+dialout:x:18:
+floppy:x:19:
+video:x:28:
+audio:x:29:
+tape:x:32:
+www-data:x:33:
+operator:x:37:
+utmp:x:43:
+plugdev:x:46:
+extio:x:47
```

Comment:
Let's use BR2_ROOTFS_USERS_TABLES instead of hard coding a list of groups, we'll just need to add a "extio" user with "extio" as it's default group-- all the other code here can stay the same.

https://buildroot.org/downloads/manual/manual.html#customize-users